app/views/admin/videos/new.html.haml

```diff
@@ -0,0 +1,21 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        = bootstrap_form_for [:admin, @video], class: 'form-horizontal' do |f|
+          %ul.nav.nav-tabs
+            %li
+              %a(href="") Recent Payments
+            %li.active
+              %a(href="") Add a New Video
+          %br
+          %fieldset.col-sm-6
+            .form-group
+              = f.text_field :title, label: "Title", class: "form-control"
+              = f.select :category_id, options_for_select(Category.all.map {|cat| [cat.name, cat.id]}), {}, class: 'form-control'
```

Comment:
I would push this - `options_for_select(Category.all.map {|cat| [cat.name, cat.id]})` to a helper method.